Test-network.md

```diff
@@ -139,7 +139,7 @@ The following assumes that docker and psql are already installed.
 // pull down official docker image
 $ docker pull postgres
 
-$ docker run --rm -p 5432:5432 --name postgres postgres
+$ docker run --rm -e POSTGRES_PASSWORD=storj -p 5432:5432 --name postgres postgres
```

Comment:
If we are using a password, line 157 must be updated accordingly

`$ storj-sim network setup --postgres=postgres://postgres:storj@localhost/<dbName>?sslmode=disable`

However, it's possible to run without a password as before using the environment variable `POSTGRES_HOST_AUTH_METHOD=trust`, so this command would turn to be:

`docker run --rm -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 --name postgres postgres`

To keep it as it's I would prefer to use it.

Reply:
I also prefer `AUTH_METHOD=trust`. Have updated the docs accordingly.